katzenmint.toml

```diff
@@ -2,6 +2,7 @@ TendermintConfigPath="./chain/config/config.toml"
 DBPath="./data"
 Layers=1
 MinNodesPerLayer=1
+Membership=true
```

Comment:
Except for nodes in the genesis block, membership should be initialized as false